rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -177,11 +177,11 @@ def test_exec(config):
             srv_restarted = rgw_service.restart()
 
         if config.test_aync_data_notifications:
+            set_log = "ceph config set global log_to_file true"
```

Comment:
Can we add above line using below function
ceph_conf.set_to_ceph_conf(
                "global",
                ConfigOpts.log_to_file,
                True,
            )